TracksRemoveRedundant.php

```diff
@@ -1,19 +1,18 @@
 <?php
 /*
+    This routine looks at all tracks data and deletes the data if a flight time was not during a flight.
+    This has the effect of removing location data for Gliders who are stationary on the ground.
+    
     From apache as a web page: http://<hostname>/ArchiveTracks.php
-    From a CRON Job:  php ArchiveTracks.php <route directory>
-      Example: php ArchiveTracks.php /var/www/html
+    From a CRON Job:  php TracksRemoveRudundant.php <route directory>
+      Example: php TracksRemoveRudundant.php /var/www/html
```

Comment:
Just a small note, if we named it ```$configDir``` it might be better for the script to expect to be called with the path to the config folder. E.g.``` php ArchiveTracks.php /var/www/html/config``` and use ```$db_params = require($configDir . '/database.php');``` to get to the database configuration.

Unless you envision that the scripts might need to reference other parts of the application too (maybe helper functions) in which case maybe rename ```$configDir``` to something like ```$baseDi```r or similar.